common/rpc-service.c

```diff
@@ -1535,6 +1536,7 @@ seafile_edit_repo (const char *repo_id,
                               "Changed library name or description",
                               0);
     commit->parent_id = g_strdup(parent->commit_id);
+    commit->repaired = parent->repaired;
```

Comment:
Move this to the function seaf_repo_to_commit(), which is called by all functions that creates a new commit.